test/tests/selftest/pbs_expect.py

```diff
@@ -0,0 +1,58 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2018 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# For a copy of the commercial license terms and conditions,
+# go to: (http://www.pbspro.com/UserArea/agreement.html)
+# or contact the Altair Legal Department.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software -
+# under a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.selftest import *
+
+
+class TestExpect(TestSelf):
+    """
+    Contains tests for the expect() function
+    """
+
+    def test_attribute_case(self):
+        """
+        Test that when verifying attribute list containing attribute names
+        with different case, expect() is case insensitive
+        """
+        # Create a queue
+        a = {'queue_type': 'execution'}
+        self.server.manager(MGR_CMD_CREATE, QUEUE, a, 'expressq')
+
+        # Set the Priority attribute on the queue but provide 'p' lowercase
+        # Set other attributes normally
+        a = {'enabled': 'True', 'started': 'True', 'priority': 150}
```

Comment:
Don't you have to include queue_type?  If you don't, you can't set enabled and started.

Reply:
@bhroam he is already creating the queue with queue_type at line 53

Reply:
oops, missed that.  Thanks for pointing it out.